### PR TITLE
Move websocket performance client and fix bugs

### PR DESCRIPTION
**Pull request by yush1ga**
### Motivation

A performance client for websocket proxy is in test directory now and it has some bugs.

### Modifications

- Move it into `pulsar-testclient` module.
- Fix bugs.

### Result
A performance client is able to be used by `pulsar-perf` command.

```
pulsar-perf websocket -h
```

### CODE REVIEW

File: bin/pulsar-perf [hunk 1]
```diff
@@ -79,6 +79,7 @@ where command is one of:
     monitor-brokers         Continuously receive broker data and/or load reports
     simulation-client       Run a simulation server acting as a Pulsar client
     simulation-controller   Run a simulation controller to give commands to servers
+    websocket               Run a websocket producer
```
Comment (merlimat) [inline bin/pulsar-perf]: Rename the command `websocket-producer`?

File: pulsar-testclient/src/main/java/com/yahoo/pulsar/proxy/socket/client/PerformanceClient.java [hunk 2]
```diff
@@ -62,17 +62,17 @@
         @Parameter(names = { "--conf-file" }, description = "Configuration file")
         public String confFile;
 
-        @Parameter(names = { "-u", "--proxy-url" }, description = "Pulsar Proxy URL", required = true)
+        @Parameter(names = { "-u", "--proxy-url" }, description = "Pulsar Proxy URL, e.g., \"ws://localhost:8080/\"", required = true)
         public String proxyURL;
 
         @Parameter(description = "/persistent/my-property/cluster1/my-ns/my-topic", required = true)
-        public String destination;
+        public List<String> destinations;
```
Comment (saandrews) [inline pulsar-testclient/src/main/java/com/yahoo/pulsar/proxy/socket/client/PerformanceClient.java]: If we are going to allow only one entry in the list, what is the reason to move it to a list?

Comment (yush1ga) [inline pulsar-testclient/src/main/java/com/yahoo/pulsar/proxy/socket/client/PerformanceClient.java]: JCommander did't support a single main parameter until it was update v1.72.
https://github.com/cbeust/jcommander/issues/90

Unfortunately, v1.72 is not in Maven Repository now.
https://mvnrepository.com/artifact/com.beust/jcommander

That's why, we have to define a main parameter as a `List<String>`.

File: bin/pulsar-perf (hunk 1 above)
Comment (yush1ga) [inline bin/pulsar-perf]: Fixed :)

**Pull request merged**